etc/config.xml

```diff
@@ -230,5 +230,17 @@
                 <group>adyen</group>
             </adyen_google_pay>
         </payment>
+        <dev>
+            <js>
+                <minify_exclude>
+                    <adyen_payment>/adyen</adyen_payment>
```

Comment:
I think this will also exclude any local assets that matches to `*/adyen*`, is there a better pattern to match external URLs? I've recommended `adyen.com/checkoutshopper` before but I'm not sure if it's a good match